runtime/gc_vlhgc/InterRegionRememberedSet.hpp

```diff
@@ -185,8 +189,39 @@ class MM_InterRegionRememberedSet : public MM_BaseVirtual
 	void clearFromRegionReferencesForCompactOptimized(MM_EnvironmentVLHGC* env);
 	MM_InterRegionRememberedSet(MM_HeapRegionManager *heapRegionManager);
 	
+
+	/**
+	 * Helper function isCompressedCardDirtyForPartialCollect
+	 * extended from compressedCardTable->isCompressedCardDirtyForPartialCollect(), only in case first PGC after GMP, also dirty card if card Contains No Object.
```

Comment:
comment says 'also dirty card' 
did you mean 'also remove card'?

Reply:
compressedCardTable->isCompressedCardDirtyForPartialCollect() return true if the card is dirty, extra condition if the card Contains No Object return true (the card is dirty)

Reply:
Ah, then it should say 'also _return as_ dirty'. As-is I understood you imply an _action_ of dirtying the card.